### ISSUE TITLE
World map: Hide p2p areas while in f2p world

### ISSUE THREAD

**Issue by blakegearin** [runelite/runelite#12599]
**Is your feature request related to a problem? Please describe.**
As a f2p player the world map shows me more information than I need.

**Describe the solution you'd like**
When in f2p worlds, add a toggle to hide/grey out areas on the world map that I can't access as a f2p member.

**Describe alternatives you've considered**

1. Turn off all icons outside the f2p areas as well as p2p icons in f2p areas (similar to #11599)
2. There's plenty of world map settings in RuneLite already toggle-able. Perhaps you could divide them into f2p, p2p, and a general category that applies to both, then users can easily toggle off all p2p-related settings

**Additional context**
None

**Comment by Osrepnay** [comment]
I agree, sometimes it gets confusing figuring out which parts are f2p and which are p2p

**Comment by ErikHumphrey** [comment]
Additional idea that I got reading the title: It would be cool to have a plugin that makes members' areas not render at all in the world; i.e., a plugin that makes anything not in an F2P tile not visible at all. Probably warrants a separate issue, though.

**Comment by kslop** [comment]
This is a good idea

**Comment by KivaGale** [comment]
Im thinking of trying my hand at a F2P plugin.
It would have this along with other stuff designed to make the members side disappear. 
Such as hiding members skills and members items from skill guides
